### DISCUSSION TITLE
Can I customize multiple keyword searches，like this。

### DISCUSSION THREAD

**Discussion by laiquziru** [dnGrep/dnGrep#1196] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/04c71f37-c9e7-4398-b5b7-250eb3910383)

**Comment by fade2gray** [comment, marked as answer]
You have the "Regular expression" search option checked, but you are not using a regular expression in the search field.

Try using `(666c6167)|(flag)|(abc)`

https://regex101.com/r/yOCMXL/1
